### PR TITLE
Initial server setup

### PR DESCRIPTION
**Pull request by rightway1**
Added swagger docs, updated readme, etc

### CODE REVIEW

File: server/web/templates/welcome.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>mapTrunk API</title>
```
Comment (LindaAlblas) [inline server/web/templates/welcome.html]: change to DMIS API

Comment (rightway1) [inline server/web/templates/welcome.html]: Done

File: bin/linux/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+        # Deploy develop builds to Staging environment
+        echo Deploying $VERSION to dmis-staging
+        eb use dmis-staging
+        eb deploy -l $VERSION
+
+fi
+
+# Only deploy to Prod if we're on master and testing the Prod Environment
+if [ $BRANCH == "master" ]
```
Comment (hunt3ri) [inline bin/linux/deploy.sh]: Can remove the master deploy block, as production will be on their env

File: bin/linux/integration-test.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+if [ $BRANCH == "develop" ]
+    then
+        export DMIS_ENV="staging"
+        echo Running integration tests in Develop branch against environment $DMIS_ENV
+        nosetests ./tests/server/integration --with-xunit --xunit-file ./shippable/testresults/integrationresults.xml
+        xvfb-run --server-args="-ac" behave tests/behave --junit --junit-directory ./shippable/testresults/shippable
+fi
+
+if [ $BRANCH == "master" ]
```
Comment (hunt3ri) [inline bin/linux/integration-test.sh]: Similar remove this as we don't have prod

File: bin/win/install.bat [hunk 4]
```diff
@@ -0,0 +1,5 @@
+REM Installer for DMIS on Win,
```
Comment (hunt3ri) [inline bin/win/install.bat]: can remove this (at least for now).  No specific win installer needed

File: server/config.py [hunk 5]
```diff
@@ -2,13 +2,21 @@
 
 
 class EnvironmentConfig(object):
+    pass
 
 
 class ProdConfig(EnvironmentConfig):
```
Comment (hunt3ri) [inline server/config.py]: No need for Prod config

File: bin/linux/deploy.sh (hunk 2 above)
Comment (rightway1) [inline bin/linux/deploy.sh]: Done

File: bin/linux/integration-test.sh (hunk 3 above)
Comment (rightway1) [inline bin/linux/integration-test.sh]: Done

File: bin/win/install.bat (hunk 4 above)
Comment (rightway1) [inline bin/win/install.bat]: might just leave this as a shortcut when setting up the venv for running locally

File: server/config.py (hunk 5 above)
Comment (rightway1) [inline server/config.py]: Done - added the API_DOCS_URL as well

**Pull request merged**